shared/Utils.ts

```diff
@@ -28,7 +28,11 @@ export interface CommunicationDetails {
 }
 
 export const isMosipVC = (issuer: string) => {
-  return issuer === Issuers.Mosip || issuer === Issuers.MosipOtp;
```

Comment:
Please check all other caller of isMosipVC. Adding MosipCertify might start causing issue for other callers.

Reply:
Removed certify condition